python/cuml/test/test_hdbscan.py

```diff
@@ -215,6 +262,55 @@ def test_hdbscan_cluster_patterns(dataset, nrows,
            np.sort(cuml_agg.cluster_persistence_), rtol=0.1, atol=0.1)
 
 
+@pytest.mark.parametrize('nrows', [1000])
+@pytest.mark.parametrize('dataset', dataset_names)
+@pytest.mark.parametrize('min_samples', [5, 50, 400, 800])
+@pytest.mark.parametrize('cluster_selection_epsilon', [0.0])
```

Comment:
For completeness, we should probably test this one w/ a few nonzero values as well.